tools/serverpod_cli/lib/src/generator/generator.dart

```diff
@@ -4,14 +4,15 @@ import 'package:serverpod_cli/src/generator/code_generator.dart';
 import 'package:serverpod_cli/src/util/print.dart';
 
 /// Analyze the server package and generate the code.
-Future<void> performGenerate({
+Future<bool> performGenerate({
   required bool verbose,
   bool dartFormat = true,
   String? changedFile,
   required GeneratorConfig config,
   required EndpointsAnalyzer endpointsAnalyzer,
 }) async {
   var collector = CodeGenerationCollector();
+  bool hadErrors = false;
```

Comment:
May be personal preference, but I feel like `hasErrors` is a bit of a more standard choice of naming.